demos/nvs-spi-flash/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Flash non-volatile storage demo configuration
+
+mainmenu "NVS Sample Configuration"
+
+config NVS_SAMPLE_MAX_REBOOT
```

Comment:
Let's not use `NVS_` for app-level configs, as they can be confused with actual NVS subsystem configuration values. Either `DEMO_` or `APP_` would suffice.